README.md

```diff
@@ -216,6 +216,10 @@ eu.xenit.alfresco.healthprocessor.plugin.solr-index.endpoints.solr-shard2.indexe
 
 </details>
 
+##### Solr 2.0
+When the health-processor is used for Solr index validation on Search Services 2.0 and upper it is advised to enable `eu.xenit.alfresco.healthprocessor.plugins.solr-index.check-transaction=true`.
+This property enables not only checking if the content is present in solr, but also if the content is fully up-to-date with the latest transaction.
```

Comment:
```suggestion
This property checks that the document is fully up-to-date with the latest transaction instead of only checking presence in the index in any version.
```